x-pack/filebeat/input/netflow/decoder/v9/v9.go

```diff
@@ -61,12 +68,16 @@ func (p *NetflowV9Protocol) Start() error {
 	if p.timeout != time.Duration(0) {
 		go p.Session.CleanupLoop(p.timeout, p.done)
 	}
+
+	p.cache.start(p.done, 30*time.Second, 10*time.Second)
```

Comment:
nit: maybe extract these as const defaults vars, possibly configurable if needed. 
some unit tests coverage for LRU implementation would be useful, that's where would need to be able to configure these values differently.

Reply:
nit: could be replaced with context.Context Done 
```
p.done = make(chan struct{})
```

Reply:
replaces with context.Context Done, feel to unresolve if you spot anything else